packages/rspack/src/config/zod.ts

```diff
@@ -944,10 +944,12 @@ const experiments = z.strictObject({
 	newSplitChunks: z
 		.boolean()
 		.optional()
-		.refine(_ => {
-			console.warn(
-				"`experiments.newSplitChunks` will be removed at 0.4.0. See details at https://github.com/web-infra-dev/rspack/discussions/4168"
-			);
+		.refine(val => {
+			if (val === false) {
```

Comment:
Maybe we should warn every user that sets this value explicitly no matter `true`/`false` is used?

Reply:
Is it possible to get the raw user input here?

Reply:
According to the deprecation message "`experiments.newSplitChunks` will be removed at 0.4.0.", looks like users with option `true` needs to be warned too.

Reply:
the `val` should be the raw user input.